app/_hub/kong-inc/json-threat-protection/overview/_index.md

```diff
@@ -0,0 +1,234 @@
+---
+nav_title: Overview
+---
+
+The JSON Threat Protection plugin provides security validation on various aspects of JSON to ensure that the incoming JSON in requests adheres to the security policy limits.
+
+It will check the following points:
+
+- Maximum container depth of the entire JSON
+- Maximum number of array elements
+- Maximum number of object entries
+- Maximum length of object keys
+- Maximum length of strings
+
+For example, for the following JSON:
+
+```
+{
+  "name": "Jason",
+  "age": 20,
+  "gender": "male",
+  "parents": ["Joseph", "Viva"]
+}
+```
+
+- Maximum container depth: 2
+- Maximum number of array elements: 2
+- Maximum number of object entries: 4
+- Maximum length of object keys: 7 (`parents`)
+- Maximum length of strings: 6 (`Joseph`)
+
+
+
+## Using the plugin
+
+We first create a service and route, and then create a JSON-threat-protection policy.
+
+
+
+### Create a Service
+
+```bash
+curl -i -s -X POST http://localhost:8001/services \
+  --data name=example_service \
+  --data url='http://localhost/'
+```
+
+In this example, an Nginx instance is started locally with the default port 80.
```

Comment:
User wont know what your end goal is. It needs explanation. See my comment under the 'Using the plugin'. This statement about starting nginx needs to come before we create service and route on Kong

Reply:
I just simply removed this sentence.